test/test_helper.rb

```diff
@@ -18,6 +18,8 @@
 # Shim for compatibility with older versions of MiniTest
 MiniTest::Test = MiniTest::Unit::TestCase unless defined?(MiniTest::Test)
 
+RubygemFs.mock!
```

Comment:
Do we need this enable for all of the tests? Or integration only? Mostly curious/wondering.